ambry-clustermap/src/main/java/com.github.ambry.clustermap/ClusterMapUtils.java

```diff
@@ -46,13 +54,18 @@
   static final String SEALED_STR = "SEALED";
   static final String STOPPED_REPLICAS_STR = "STOPPED";
   static final String AVAILABLE_STR = "AVAILABLE";
+  static final String READ_ONLY_STR = "RO";
```

Comment:
`READ_ONLY_STR` and `SEALED_STR` mean the same thing?

Reply:
The `SEALED_STR` is used for sealed list in InstanceConfig, `READ_ONLY_STR` and `READ_WRITE_STR` are used for the partition override infos. Although they have the same meaning, they are used in different places. For partition override map, it's better to make string concise and same length with `READ_WRITE_STR`. 
```"0": {
            "state": "RO"
        },
        "1": {
            "state": "RW"
        },```